app/server/utils/metrics/metric.py

```diff
@@ -66,6 +66,16 @@ def execute_query(self, user_filters: dict = None, date_filters_dict=None, enabl
 
         if self.is_timeseries:
             result = {}
+            if self.value_type not in VALUE_TYPES:
+                raise Exception(f'{self.value_type} not a valid metric type!')
+            result['type'] = {
+                'type': self.value_type,
+                'display_decimals': 2 if self.value_type == COUNT_AVERAGE else 0
```

Comment:
`change this to self.token.display_decimals once #251 is merged` also?

Reply:
Yep! Added according comment

Reply:
This is for `COUNT_AVERAGE` (so like Volume Per User). It's set to `token.display_decimals` on l78 for anything involving a token